src/components/store/AddCartButton/style.module.scss

```
@@ -0,0 +1,70 @@
+@use 'src/styles/vars.scss' as vars;
+
+.addCartGroup {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+
+  p {
+    color: var(--theme-text-on-background-1);
+    font-size: vars.$general-heading-4;
+  }
+
+  .buttonRow {
+    align-items: flex-end;
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+  }
+
+  .quantityColumn {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    @media (max-width: vars.$breakpoint-md) {
+      gap: 0.75rem;
+    }
+
+    div > select {
+      font-size: 1rem;
+      font-size: 1.25rem;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 1.25rem;
+      line-height: 200%; /* 2.5rem */
+
+      text-align: center;
+    }
+  }
+
+  .error {
+    color: vars.$danger-1;
+  }
+
+  .valid {
+    color: vars.$success-1;
+  }
+
+  .button {
+    border-radius: 10px;
+    font-size: vars.$input-text-medium;
+    font-weight: bold;
+    margin: 0.2rem 0;
+    padding: 0.3rem 1rem;
+    word-break: break-word;
+  }
+
+  .buttonInStock {
+    background-color: var(--theme-primary-2);
+    color: var(--theme-text-on-background1);
+  }
+
+  .buttonNoStock {
+    background-color: var(--theme-background);
+    border: solid;
+    border-color: var(--theme-accent-line-2);
+
+    border-width: 1px;
```

You could use the shorthand here

```suggestion
    border: 1px solid var(--theme-accent-line-2);
```